packages/app-explorer/src/index.css

```diff
@@ -61,3 +61,14 @@
 .explorer--query {
   width: 20em;
 }
+
+details[open] > summary {
```

Comment:
Would suggest moving this to `packages/ui-app/styles/app.css` since it will apply everywhere the details/summary combination is used. (Not only in the explorer app)

Reply:
ok